### PR TITLE
Todo structure

### PR DESCRIPTION
**Pull request by Haadiiii**
## Todo List structure Implementation ⏬ 👍 

In this project we : 

- Added Webpack. 👍 
- Added `index.js` file 👍 
- Added Modules js for `js` file 👍 
- Added `Array of Objects` for lists 👍 
- Implement `ES6` 👍 
- Added `index.CSS` style. 👍 
- Added `index.Html` File. 👍 
- Resolved `Linters` 👍 
- Added `README.md` FIle 👍 

😄

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  box-shadow: 2px solid white;
+}
+
+#title {
+  text-align: center;
+  width: 20vw;
+  margin-left: -3rem;
+  font-size: 24px;
+  padding: 0.3rem 0;
+  margin-top: 1rem;
```
Comment (phelian23) [inline src/index.css:31]: - [ ] The design should be similar to the design example. kindly make the design look as much as possible as the one in the video.
Your design
design

Template design
figma

File: modules/showlist.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const showList = () => {
+  let str = "";
+  task.map((tasks) => {
+    const { discription, completed, index } = tasks;
+    str += `
+      <label>${index}</label><br>
+      <label>${discription}</label><br>
+      <label>${completed}</label><br>
+      <hr>
```
Comment (phelian23) [inline modules/showlist.js:12]: - [ ] In the UI, only the description is displayed, the index and completed are used for other functions. kindly fix it

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Haadiiii,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/showlist.js (hunk 2 above)
Comment (Haadiiii) [reply modules/showlist.js]: done 👍

Comment (Haadiiii) [reply modules/showlist.js]: resolved 👍

**Review by Lordkaito — APPROVED** [review]
Hi @Haadiiii, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**